src/senaite/jsonapi/dataproviders.py

```diff
@@ -163,6 +158,16 @@ def _x_get_parent_path(self):
         return "/".join(path[:-1])
 
 
+    def _x_get_physical_path(self):
+        """Generate the physical path
+        """
+        path = self.context.getPath()
+        portal_path = api.get_path(api.get_portal())
+        if portal_path not in path:
+            return "{}/{}".format(portal_path, path)
+        return "/".join(path)
```

Comment:
Probably you need to update `_x_get_parent_path` as well, e.g. by using this method instead of `path = self.context.getPath()`